include/CL/sycl/parallelism/detail/parallelism.hpp

```diff
@@ -300,7 +300,7 @@ void parallel_for_workitem(const group<Dimensions> &g,
   // With OMP, one task is created for each work-item in the group
 
   range<Dimensions> l_r = g.get_nd_range().get_local();
-  int tot = l_r.get(0);
+  std::size_t tot = l_r.get(0);
```

Comment:
MSVC is very picky about signed/unsigned differences, especially when it comes to assigning.